proof/refine/X64/ArchArchAcc_R.thy

```diff
@@ -35,6 +30,181 @@ lemma asid_low_bits [simp]:
   "asidLowBits = asid_low_bits"
   by (simp add: asid_low_bits_def asidLowBits_def)
 
+lemma pspace_aligned_cross[ArchAcc_R_assms]:
+  "\<lbrakk> pspace_aligned s; pspace_relation (kheap s) (ksPSpace s') \<rbrakk> \<Longrightarrow> pspace_aligned' s'"
```

Comment:
I was a bit surprised to see this being moved in this commit. I guess it was here in the other arches but X64 had it in ArchKHeap? Very minor but could be worth mentioning in the commit message.

Reply:
Exactly, it was migrated by someone for X64, but then it turned out here that it didn't have to be migrated.

- [ ] add to arch-split commit message